exchange/factory_test.go

```diff
@@ -39,9 +39,10 @@ func eraseSignature(sxg []byte) []byte {
 func TestFactory(t *testing.T) {
 	// Initialize Factory with the parameters given to gen-signedexchange.
 	factory := exchange.NewFactory(exchange.Config{
-		CertChain:  certtest.ReadCertChainFile("../testdata/certs/test.cbor"),
-		CertURL:    urlutil.MustParse("https://example.org/cert.cbor"),
-		PrivateKey: certtest.ReadPrivateKeyFile("../testdata/certs/test.key"),
+		MIRecordSize: 4096,
```

Comment:
Would it be a good idea to add a test that uses the DefaultMIRecordSize you just set?

Reply:
Updated the testdata with the new DefaultMIRecordSize just in case. I'm not sure if it's worth having separate tests with a non-default record size.